README.md

```diff
@@ -1,188 +1,194 @@
-
-## Common Use Cases
-
-- **Speed up prototype to production.**
-  Scale code written and tested locally to a distributed system or cloud SQL engine with minimal rewrites.
-- **Boost performance of existing Python or pandas code.**
-  For example a general rule of thumb for pandas is "Have 5 to 10 times as much RAM as the size of your dataset".
-  When a dataset exceeds this rule using in-memory frameworks like pandas can be slow.
-  Instead, using Ibis will significantly speed up your workflows because of its deferred execution.
-  Ibis also empowers you to switch to a faster database engine, without changing much of your code.
-- **Get rid of long, error-prone, `f`-strings.**
-  Ibis provides one syntax for multiple query engines and dataframe APIs that lets you avoid learning new flavors of SQL or other framework-specific code.
-  Learn the syntax once and use that syntax anywhere.
+Ibis is the portable Python dataframe library:
 
-## Backends
-
-Ibis acts as a universal frontend to the following systems:
-
-- [Apache Arrow DataFusion](https://ibis-project.org/backends/datafusion/) (experimental)
-- [Apache Druid](https://ibis-project.org/backends/druid/) (experimental)
-- [Apache Flink](https://ibis-project.org/backends/flink)
-- [Apache Impala](https://ibis-project.org/backends/impala/)
-- [Apache PySpark](https://ibis-project.org/backends/pyspark/)
-- [BigQuery](https://ibis-project.org/backends/bigquery/)
-- [ClickHouse](https://ibis-project.org/backends/clickhouse/)
-- [Dask](https://ibis-project.org/backends/dask/)
-- [DuckDB](https://ibis-project.org/backends/duckdb/)
-- [Exasol](https://ibis-project.org/backends/exasol) (experimental)
-- [HeavyAI](https://github.com/heavyai/ibis-heavyai)
-- [MySQL](https://ibis-project.org/backends/mysql/)
-- [Oracle](https://ibis-project.org/backends/oracle/) (experimental)
-- [Pandas](https://ibis-project.org/backends/pandas/)
-- [Polars](https://ibis-project.org/backends/polars/) (experimental)
-- [PostgreSQL](https://ibis-project.org/backends/postgresql/)
-- [SQL Server](https://ibis-project.org/backends/mssql/)
-- [SQLite](https://ibis-project.org/backends/sqlite/)
-- [Snowflake](https://ibis-project.org/backends/snowflake) (experimental)
-- [Trino](https://ibis-project.org/backends/trino/) (experimental)
+- Fast local dataframes with the default DuckDB backend
+- Lazy (by default) or interactive execution
+- [Mix Python dataframe and SQL code](#python--sql-better-together)
+- Use the same dataframe API for [20+ backends](#backends)
+- Iterate locally and deploy remotely by [changing a single line of code](#portability)
 
-The list of supported backends is continuously growing. Anyone can get involved
-in adding new ones! Learn more about contributing to ibis in our contributing
-documentation at https://github.com/ibis-project/ibis/blob/main/docs/CONTRIBUTING.md
+See the documentation on ["Why Ibis?"](https://ibis-project.org/why) to learn more.
```

Comment:
should we add a graph in readme to explain what ibis is conceptually?

Reply:
yes but I don't have a good one

there is a nice one here: https://voltrondata.com/resources/open-source-standards

we'd need to de-VoDa-fy it. perhaps we can engage the VoDa design team after GTC? graphs/diagrams would also be good in general for the website

I suggest we don't block this update on having a graph though (there isn't one currently)

Reply:
If anything, that Sankey diagram seems like a reasonable start. Agree that this shouldn't block.

Reply:
added Sankey